src/pages/workspaces/List.js

```diff
@@ -128,6 +132,10 @@ export const WorkspaceList = () => {
     loadFeatured()
   })
 
+  useEffect(() => {
+    setLocalPref(persistenceId, { recentlyViewedOpen })
+  }, [recentlyViewedOpen]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Unfortunately, `react-hooks/exhaustive-deps` doesn't allow ignoring specific dependencies. That being the case, in my opinion, it's preferable to leave this rule enabled and add the unnecessary `persistenceId` dependency rather than chance missing a dependency if the effect function is changed later.

Reply:
Updated in 0c10d40fb89481394ebfac66175db863a634bef1